package.json

```diff
@@ -30,6 +30,7 @@
     "docker:local:start": "docker run -d --restart=unless-stopped -p 80:80 -p 443:443 -e CATTLE_BOOTSTRAP_PASSWORD=password -e CATTLE_PASSWORD_MIN_LENGTH=3 --name cypress --privileged rancher/rancher:v2.7-head",
     "docker:local:stop": "docker kill cypress || docker rm cypress || true",
     "build": "./node_modules/.bin/nuxt build --devtools",
+    "build:ci": "node --max-old-space-size=8192 ./node_modules/.bin/nuxt build",
```

Comment:
Why is `--max-old-space-size=8192` required?

Reply:
We do not use nuxt anymore as well, so it should be something like
```
    "build: ci": "NODE_OPTIONS=--max_old_space_size=8192 ./node_modules/.bin/vue-cli-service build",
```

Although we had the same issue in build, so maybe we just stick to the current value